dist/practice.js

```diff
@@ -1052,10 +1052,9 @@ var makeUnit = (function() {
 
 var getEnhancement = function(card, s, base) {
     var enhancements = card.enhanced;
-    var enhanced = (enhancements ? (enhancements[s] || 0) : 0);
-    if (enhanced < 0) {
-        enhanced = Math.ceil(base * -enhanced);
-    }
+    var e = { 'x': 0, 'mult': 0 }; // Default value
```

Comment:
**nit:** Prefer non-quoted property keys for consistency with the rest of the codebase.

Ditto throughout.